cscs-checks/microbenchmarks/cpu/simd/vc.py

```diff
@@ -31,14 +31,19 @@ class VcSimdTest(rfm.RegressionTest):
 
     def __init__(self):
         self.valid_systems = ['*']
-        self.valid_prog_environs = ['builtin']
+        if self.current_system.name in ['eiger', 'pilatus']:
+            self.valid_prog_environs = ['cpeGNU']
+        else:
+            self.valid_prog_environs = ['PrgEnv-gnu']
+
         self.descr = 'finitediff example'
         self.build_system = 'SingleSource'
         self.testname = 'finitediff'
         src_url = 'https://raw.githubusercontent.com/VcDevel/Vc/1.4/examples'
         src1 = '%s/tsc.h' % src_url
         src2 = '%s/finitediff/main.cpp' % src_url
         self.prebuild_cmds = [
+            'module list',
```

Comment:
@jgphpc Isn't this test removed by #1966 ?

Reply:
I have removed `vc.py` to resolve the conflict with master